example.init.yaml

```diff
@@ -140,20 +160,25 @@ tls_config:
   issuer_type: "staging"
   email: "your@email.com"
 
-
-  ## Select DNS web service between Amazon Route 53 (route53) and Google Cloud DNS (clouddns)	
-  # by uncommenting the required option	
+  ## Select DNS web service between Amazon Route 53 (route53) and Google Cloud DNS (clouddns)
+  # by uncommenting the required option
 
   ### DigitalOcean
   dns_service: digitalocean
-
-
-  ### Google Cloud DNS	
-  # dns_service: clouddns	
+  ### Google Cloud DNS
+  # dns_service: clouddns
   # project_id: "my-openfaas-cloud"
-
   ### AWS Route53
   # dns_service: route53
   # region: us-east-1
   # access_key_id: ASYAKIUJE8AYRQQ7DU3M
 
+##  Gateway configuration
+### Basic authentication is turned on by default so if you did not explicitly disabled it
+### keep this as is
+basic_auth: true
+
+##  Internal trust
+### This is security feature which is enabled by default for OFC to ensure that only functions
+### inside your OFC instance can call eachother
+internal_trust: true
```

Comment:
`internal_trust: true`

Not sure this should be optional.

Reply:
I will remove this, I will create `default` filter for those cases

Reply:
> Not sure this should be optional.
You are totally right 👍  I just wanted to check the review

Reply:
Removed 👍